cypress/support/step-definitions/radio-button-steps.js

```diff
@@ -109,3 +116,29 @@ Then('{string} RadioButton label width is set to {string}', (radioButtonName, wi
 When('I set RadioButton {word} {word} slider to {int}', (propertyName, text, width) => {
   setSlidebar(labelWidthSliderByName(propertyName, text), width);
 });
+
+Then('Then RadioButton are inline', () => {
```

Comment:
We don't need `Then` here, because BDD step itself is called `Then`
```suggestion
Then('RadioButton are inline', () => {
```